src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts

```diff
@@ -46,6 +57,15 @@ export class UbsBaseSidebarComponent implements AfterViewInit, OnDestroy {
     }
   }
 
+  @HostListener('window:resize', ['$event'])
+  onResize(event) {
+    if (event.target.innerWidth < 992 || window.innerWidth < 992) {
+      if (this.drawer) {
```

Comment:
don't use the nested "if" here
if ((event.target.innerWidth < 992 || window.innerWidth < 992) && this.drawer ) {